static/styles/explorer.scss

```diff
@@ -570,6 +570,10 @@ kbd {
     height: 20px !important;
 }
 
+li.lm_tab.lm_active {
```

Comment:
is this intended too?

Reply:
Yes. Again, should have been a different PR